float32 x_center
float32 y_center
float32 area
float32 rotation_angle
float32 cam_height
float32 cam_width